app/yaba.yaml

```diff
@@ -737,7 +737,82 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /cultivars_whitespace_test:
```

Comment:
Should this be removed from the commit since it's test-only; don't want end users to do this